tests/test_config_entries.py

```diff
@@ -2818,7 +2818,9 @@ def _record_load(event: Event) -> None:
         nonlocal load_events
         load_events.append(event)
 
-    listener = hass.bus.async_listen(EVENT_COMPONENT_LOADED, _record_load)
+    listener = hass.bus.async_listen(
+        EVENT_COMPONENT_LOADED, _record_load, run_immediately=True
+    )
```

Comment:
I think this whole method can be replaced by the use of `tests.common.async_capture_events`

Reply:
5cb4c9abeff8291d5aa3b8d754dc9876267094f5